types/n3/index.d.ts

```diff
@@ -246,6 +246,10 @@ export class StreamWriter<Q extends RDF.BaseQuad = RDF.Quad> extends stream.Tran
     import(stream: RDF.Stream<Q>): EventEmitter;
 }
 
```

Comment:
I think this should be parameterised similar to the Store to permit the use of RDF.BaseQuad. It think it should also be permitted to supply BaseQuad in the constructor.

Reply:
Done